CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Fixed a crash when `viewer.zoomTo` or `viewer.flyTo` were called immediately before or during a scene morph [#2775](https://github.com/AnalyticalGraphicsInc/cesium/issues/2775).
 * Fixed an issue where `Camera` functions would throw an exception if used from within a `Scene.morphComplete` callback [#2776](https://github.com/AnalyticalGraphicsInc/cesium/issues/2776).
 * `Model` can now load Binary glTF from a Uint8Array.
+* Added `UrlTemplateImageryProvider`.
```

Comment:
This is a great feature.  Thanks again @kaktus40 and @kring.  Are there any specific examples we can mention here or even in the 1.11 release blog about what new services this allows Cesium to connect to?  Anything beyond the `@examples` for the type?